From 97a586e868f0c846625f5635ed5d83c749ee485e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Oliver=20St=C3=A4bler?= <oliver.staebler@bytesatwork.ch>
Date: Thu, 2 Nov 2017 09:59:53 +0100
Subject: [PATCH] ARM: dts: am43xx: Add missing pin names

Pins 0x280, 0x288, and 0x28C are not documented in "AM437x ARM
CortexTM-A9 Processors Technical Reference Manual" as of June 2017.
---
 include/dt-bindings/pinctrl/am437x-names.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/dt-bindings/pinctrl/am437x-names.h b/include/dt-bindings/pinctrl/am437x-names.h
index 16523e1246ab..77a7eaf6086f 100644
--- a/include/dt-bindings/pinctrl/am437x-names.h
+++ b/include/dt-bindings/pinctrl/am437x-names.h
@@ -116,6 +116,7 @@
 #define	PIN_CAM0_HD		0x1B0
 #define	PIN_CAM0_VD		0x1B4
 #define	PIN_CAM0_FIELD		0x1B8
+#define	PIN_CAM0_WEN		0x1BC
 #define	PIN_CAM0_PCLK		0x1C0
 #define	PIN_CAM0_DATA8		0x1C4
 #define	PIN_CAM0_DATA9		0x1C8
@@ -162,6 +163,7 @@
 #define	PIN_SPI2_CS0		0x26C
 #define	PIN_XDMA_EVT_INTR0	0x270
 #define	PIN_XDMA_EVT_INTR1	0x274
+#define	PIN_CLKREQ		0x278
 #define	PIN_NRESETIN_OUT	0x27C
 #define	PIN_NNMI		0x284
 #define	PIN_TMS			0x290
-- 
2.11.0

